cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.random().toString().slice(2, 6);
+  const username = faker.internet.userName() + '_' + randomNumber;
+  const email = `${username}@mail.com`;
+  const password = '12345Qwert!';
```

Comment:
It's better to generate a random password for each user. This can help to increase the coverage of your tests, as you will be testing the system with different data every time. For example, you can use the `faker` library to generate a random password. Here is an example of how you can do it:
javascript
const password = faker.internet.password();
```
Remember, the goal of using random data in tests is to mimic the unpredictability of data that your application will face in the real world.